KifuSF/UIStickyView.swift

```diff
@@ -48,38 +49,39 @@ class KFVSticky<T: UIView>: UIView, UIConfigurable {
         addSubview(contentView)
         configureLayout()
     }
-    
-    func configureLayout() {
-        contentView.translatesAutoresizingMaskIntoConstraints = false
-        
-        layoutConstraints = [
-        contentView.autoPinEdge(toSuperviewEdge: .top,
-                                withInset: (stickySide == .top) ? offset : 0,
-                                relation: (stickySide?.opositeSide() == .top) ? .greaterThanOrEqual : .equal),
-        contentView.autoPinEdge(toSuperviewEdge: .leading,
-                                withInset: (stickySide == .leading) ? offset : 0,
-                                relation: (stickySide?.opositeSide() == .leading) ? .greaterThanOrEqual : .equal),
-        contentView.autoPinEdge(toSuperviewEdge: .trailing,
-                                withInset: (stickySide == .trailing) ? offset : 0,
-                                relation: (stickySide?.opositeSide() == .trailing) ? .greaterThanOrEqual : .equal),
-        contentView.autoPinEdge(toSuperviewEdge: .bottom,
-                                withInset: (stickySide == .bottom) ? offset : 0,
-                                relation: (stickySide?.opositeSide() == .bottom) ? .greaterThanOrEqual : .equal)
-        ]
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
     }
     
-    
+    //MARK: I have no idea who wrote this code 🤷‍♂️... definitly not me 🤥
+    //FIXED: removed the ugly function, no need for somebody else to see it
```

Comment:
Is this comment current? or outdated

Reply:
Outdated, let's just leave it there, it's part of our history :]